### DISCUSSION TITLE
Icon for Q&A is a bit difficult to understand

### DISCUSSION THREAD

**Discussion by actual-nh** [community/community#2933] [Discussions] [answered]
It looks like a symbol for prayer, which does not seem right. Perhaps a question mark above a handshake?

**Comment by becca** [comment, marked as answer]
Thank you for this feedback. 

Right now you can update the emoji for a category (so long as you have the right permissions) and we'll be looking into updating the default emoji for this category!

**Reply by Andre601** [reply]
While we're at the topic about emojis do I wonder if you would ever consider to switch to a different emoji-set like Tweemoji, as I believe the current one is... "limited" and at least for me feels dated in terms of some looks.

**Comment by Andre601** [comment]
Considering that the "icon" is essentially an emoji is there not much you can do in terms of customization, as GitHub uses a specific set of emojis.
This is nothing like SVG icons you can find on sites like fontawesome...

**Comment by jmario74** [comment]
from my point of view it looks like a dick. why not just a question mark.
